Assets/Scripts/Properties/Properties.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+/// <summary>
+/// Class that mimics the behavior of [Java properties](https://docs.oracle.com/en/java/javase/17/docs/api//java.base/java/util/Properties.html).
```

Comment:
Does that link format work for C# comments?

Reply:
yes

Reply:
yes